libs/domains/clusters/feature/src/lib/hooks/use-edit-cloud-provider-info/use-edit-cloud-provider-info.ts

```diff
@@ -5,7 +5,7 @@ import { useDeployCluster } from '../use-deploy-cluster/use-deploy-cluster'
 
 export function useEditCloudProviderInfo({ silently = false } = {}) {
   const queryClient = useQueryClient()
-  const { mutateAsync: deployCluster } = useDeployCluster()
+  const { mutate: deployCluster } = useDeployCluster()
```

Comment:
Avoid JS error if we didn't add try catch in the code. Prefer to use `mutate` instead of `mutateAsync`